dlgr/griduniverse/experiment.py

```diff
@@ -978,13 +991,32 @@ class Item:
     id: int = field(default_factory=lambda: uuid.uuid4())
     creation_timestamp: float = field(default_factory=time.time)
     position: tuple = (0, 0)
+    remaining_uses: int = 1
 
     def __post_init__(self):
         object.__setattr__(self, "item_id", self.item_config["item_id"])
```

Comment:
It's only required if we want to freeze those attributes, which probably isn't a bad idea. I'll add a comment. We probably don't need it, but I got a little obsessed with making it work. I do think there's a bug here though because the `__post_init__` call will reset remaining_uses if a value was passed in the constructor. I'll fix that and add a little documentation.